script/os-autoinst-setup-multi-machine

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+set -euo pipefail
+instances="${instances:-20}"
+bridge="${bridge:-"br1"}"
+
+zypper -n in openvswitch os-autoinst-openvswitch firewalld
+ovs-vsctl add-br $bridge
+echo "OS_AUTOINST_USE_BRIDGE=$bridge" > /etc/sysconfig/os-autoinst-openvswitch
+cat > /etc/sysconfig/network/ifcfg-$bridge <<EOF
+BOOTPROTO='static'
+IPADDR='10.0.2.2/15'
+STARTMODE='auto'
+ZONE=trusted
+OVS_BRIDGE='yes'
+PRE_UP_SCRIPT="wicked:gre_tunnel_preup.sh"
+OVS_BRIDGE_PORT_DEVICE_0='tap0'
+EOF
+cat > /etc/sysconfig/network/ifcfg-tap0 <<EOF
+BOOTPROTO='none'
+IPADDR=''
+NETMASK=''
+PREFIXLEN=''
+STARTMODE='auto'
+TUNNEL='tap'
+TUNNEL_SET_GROUP='nogroup'
+TUNNEL_SET_OWNER='_openqa-worker'
+EOF
+for i in $(seq 1 $instances; seq 64 $((64+$instances)); seq 128 $((128+$instances))); do ln -sf ifcfg-tap0 /etc/sysconfig/network/ifcfg-tap$i && echo "OVS_BRIDGE_PORT_DEVICE_$i='tap$i'" >> /etc/sysconfig/network/ifcfg-$bridge; done
+cat > /etc/firewalld/zones/trusted.xml <<EOF
+<?xml version="1.0" encoding="utf-8"?>
+<zone target="ACCEPT">
+  <short>Trusted</short>
+  <description>All network connections are accepted.</description>
+  <service name="isotovideo"/>
+  <interface name="$bridge"/>
+  <interface name="ovs-system"/>
+  <interface name="eth0"/>
```

Comment:
The interface should be configurable, similar to the bridge.

Reply:
done